### PR TITLE
[Docs] Add the link and introduction of mmflow in README

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add the link and introduction of [mmflow](https://github.com/open-mmlab/mmflow) in README.

## Modification

- README.md
- README_zh-CN.md
- docs/conf.py
- docs_zh_CN/conf.py

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects, like MMDet or MMCls.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs_zh_CN/conf.py [hunk 1]
```diff
@@ -117,18 +116,22 @@
...
+                },
                 {
                     'name': 'MMAction2',
                     'url': 'https://mmaction2.readthedocs.io/zh_CN/latest/',
                 },
                 {
                     'name': 'MMClassification',
-                    'url':
-                    'https://mmclassification.readthedocs.io/zh_CN/latest/',
+                    'url': 'https://github.com/open-mmlab/mmclassification',
```
Comment (mzr1996) [inline docs_zh_CN/conf.py:129]: Why?

File: docs_zh_CN/conf.py [hunk 2]
```diff
@@ -117,18 +116,22 @@
...
                 },
                 {
                     'name': 'MMDetection',
-                    'url': 'https://mmdetection.readthedocs.io/zh_CN/latest/',
+                    'url':
+                    'https://mmclassification.readthedocs.io/zh_CN/latest/',
```
Comment (teamwong111) [inline docs_zh_CN/conf.py:134]: why?

File: README.md [hunk 3]
```diff
@@ -19,8 +21,9 @@ research projects as below:
 - [MMTracking](https://github.com/open-mmlab/mmtracking): OpenMMLab video perception toolbox and benchmark.
 - [MMPose](https://github.com/open-mmlab/mmpose): OpenMMLab pose estimation toolbox and benchmark.
 - [MMEditing](https://github.com/open-mmlab/mmediting): OpenMMLab image and video editing toolbox.
-- [MMOCR](https://github.com/open-mmlab/mmocr): OpenMMLab text detection, recognition and understanding toolbox.
+- [MMOCR](https://github.com/open-mmlab/mmocr): A Comprehensive Toolbox for Text Detection, Recognition and Understanding.
```
Comment (teamwong111) [inline README.md:24]: I think it's better to start with `OpenMMLab`

File: README_zh-CN.md [hunk 4]
```diff
@@ -10,16 +10,19 @@
 
 MMCV 是一个面向计算机视觉的基础库，它支持了很多开源项目，例如：
 
-- [MMClassification](https://github.com/open-mmlab/mmclassification): OpenMMLab 图像分类工具箱
-- [MMDetection](https://github.com/open-mmlab/mmdetection): OpenMMLab 目标检测工具箱
-- [MMDetection3D](https://github.com/open-mmlab/mmdetection3d): OpenMMLab 新一代通用 3D 目标检测平台
-- [MMSegmentation](https://github.com/open-mmlab/mmsegmentation): OpenMMLab 语义分割工具箱
-- [MMAction2](https://github.com/open-mmlab/mmaction2): OpenMMLab 新一代视频理解工具箱
+- [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab 计算机视觉基础库
+- [MIM](https://github.com/open-mmlab/mim): MIM 是 OpenMMLab 项目、算法、模型的统一入口
```
Comment (teamwong111) [inline README_zh-CN.md:14]:
```suggestion
- [MIM](https://github.com/open-mmlab/mim): OpenMMLab 项目、算法、模型的统一入口
```

File: README.md [hunk 5]
```diff
@@ -11,6 +11,8 @@ English | [简体中文](README_zh-CN.md)
 MMCV is a foundational library for computer vision research and supports many
 research projects as below:
 
+- [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab foundational library for computer vision.
+- [MIM](https://github.com/open-mmlab/mim): MIM Installs OpenMMLab Packages.
```
Comment (teamwong111) [inline README.md:15]:
```suggestion
- [MIM](https://github.com/open-mmlab/mim): OpenMMLab Package Manager.
```

File: docs_zh_CN/conf.py (hunk 2 above)
Comment (zhouzaida) [reply docs_zh_CN/conf.py:134]: I will fix it.

File: docs_zh_CN/conf.py (hunk 1 above)
Comment (zhouzaida) [reply docs_zh_CN/conf.py:129]: I will fix it.

File: README_zh-CN.md (hunk 4 above)
Comment (zhouzaida) [reply README_zh-CN.md:14]: done

File: docs_zh_CN/conf.py (hunk 2 above)
Comment (zhouzaida) [reply docs_zh_CN/conf.py:134]: done

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**